PowerEditor/src/NppCommands.cpp

```diff
@@ -140,22 +140,22 @@ void Notepad_plus::command(int id)
 			TCHAR currentDir[CURRENTWORD_MAXLENGTH];
 			::SendMessage(_pPublicInterface->getHSelf(), NPPM_GETFULLCURRENTPATH, CURRENTWORD_MAXLENGTH, reinterpret_cast<LPARAM>(currentFile));
 			::SendMessage(_pPublicInterface->getHSelf(), NPPM_GETCURRENTDIRECTORY, CURRENTWORD_MAXLENGTH, reinterpret_cast<LPARAM>(currentDir));
-	

```

Comment:
Please restore all unnecessary formatting changes.

Reply:
Done. I have just now pushed the commit "Restored white space changes".